noxfile.py

```diff
@@ -220,7 +154,8 @@ def pyright(session: nox.Session):
 )
 def test(session: nox.Session, extras: List[str]):
     """Run tests."""
-    install(session, "dev", *extras)
+    extras = [f"-G {extra}" for extra in extras]
+    session.run_always("pdm", "install", "-dG", "test", "-dG", "typing", *extras, external=True)
```

Comment:
I don't think this will work once we enable extras:
```py
nox > Running session test-3.10(extras=['speed'])
nox > Creating virtual environment (virtualenv) using python3.10 in .nox/test-3-10-extras-speed
nox > pdm install -dG test -dG typing '-G speed'
Ignoring non-existing groups:  speed
```

Reply:
Odd, those extra quotes shouldn't be there.

Reply:
nox uses `popen` which doesn't do shell splitting, so that's expected as it calls `.run_always(..., "-dG", "typing", "-G speed", ...)`.

Reply:
https://github.com/DisnakeDev/disnake/pull/836/commits/32418ba307a9f561bfe3426d069142b11fbd387d

Reply:
https://github.com/DisnakeDev/disnake/pull/836/commits/dd7724e12d22168e9f10b7da06fdc7cb45cbbdb4

Reply:
https://github.com/DisnakeDev/disnake/pull/836/commits/83917a14eabd76158fa63b91e4c9c2a5b23dfea0